common/lib/api-client/index.js

```diff
@@ -18,7 +18,7 @@ const models = require('./models')
 
 let instance
 
-module.exports = function () {
+module.exports = function (req) {
   if (instance) {
```

Comment:
Won't this mean that whatever is passed as `req` on first instantiation will get cached/used for ever?

Reply:
Probably :D Didn't fix that, we don't need to check anymore, it will get instantiated on the req and that's what we want. Nice catch, will fix!

Reply:
```js
const cache = {}

module.exports = function (req = {}) {
    const {transactionId} = req
    if (cache[transactionId]) { return cache[transactionId] }
    const instance = new JsonApi(
    ....
    cache[transactionId] = instance
    return instance
}
````

Reply:
Ah - ignore that bit ^^^

Reply:
Removed the instance bit. Tests don't like it yet, but they will!